modules/gapi/perf/render/gapi_render_perf_tests_inl.hpp

```diff
@@ -0,0 +1,1144 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_RENDER_PERF_TESTS_INL_HPP
+#define OPENCV_GAPI_RENDER_PERF_TESTS_INL_HPP
+
+#include <iostream>
+
+#include "../common/gapi_render_perf_tests.hpp"
+
+#include "../../src/api/render_priv.hpp"
+
+namespace opencv_test
+{
+  using namespace perf;
+
+  namespace {
+    void create_rand_mats(const cv::Size& size, MatType type, cv::Mat& ref_mat, cv::Mat& gapi_mat) {
+        ref_mat.create(size, type);
+        cv::randu(ref_mat, cv::Scalar::all(0), cv::Scalar::all(255));
+        ref_mat.copyTo(gapi_mat);
+    };
+
+    // TODO: remove this code duplication with /gapi/test/common/gapi_render_tests.hpp
```

Comment:
I think it's better to use `FIXME` instead of `TODO`

Reply:
Fixed